addons/html_builder/static/src/core/builder_options_plugin.js

```diff
@@ -75,6 +76,26 @@ export class BuilderOptionsPlugin extends Plugin {
         this.updateContainers(ev.target);
     }
 
+    getReloadSelector(editingElement) {
+        for (let i = this.lastContainers.length - 1; i >= 0; i--) {
+            for (const option of this.lastContainers[i].options) {
+                if (option.reloadTarget) {
+                    return option.selector;
+                }
+            }
+        }
```

Comment:
```suggestion
        for (const container of [...this.lastContainers].reverse()) {
            for (const option of container.options) {
                if (option.reloadTarget) {
                    return option.selector;
                }
            }
        }
```

Just a little more readable